website/docs/examples/kubernetes/additionalPrinterColumns.md

```diff
@@ -0,0 +1,80 @@
+---
+sidebar_position: 20
+---
+
+# Additional Printer Columns
+
+You can define `additionalPrinterColumns` for the created CRD through the ResourceGraphDefinition by setting them on `spec.schema.additionalPrinterColumns`.
+
+An example of such an RGD:
```

Comment:
hyperlink to the examples/kubernetes/additionalPrinterColumns instead of inlining here.